### ISSUE TITLE
1m_received_traffic_overflow alarm is faulty on 10G or 40G network interfaces

### ISSUE THREAD

**Issue by ktsaou** [netdata/netdata#4577]
##### Bug report summary

I received an email from a user saying that the alarm reports values above 100% when the interface is 10G/40G.

```sh
# cat /sys/class/net/eth2/speed
10000
```

![image](https://user-images.githubusercontent.com/2662304/48126933-2db57b80-e28b-11e8-996c-8240cd3e621a.png)

```sh
# ethtool eth2 | grep -E 'Speed|Port'
Speed: 10000Mb/s
Port: FIBRE

# lspci | grep Eth | grep 10
01:00.0 Ethernet controller: Intel Corporation Ethernet Controller
10-Gigabit X540-AT2 (rev 01)
01:00.1 Ethernet controller: Intel Corporation Ethernet Controller
10-Gigabit X540-AT2 (rev 01)
05:00.0 Ethernet controller: Intel Corporation 82599ES 10-Gigabit
SFI/SFP+ Network Connection (rev 01)
05:00.1 Ethernet controller: Intel Corporation 82599ES 10-Gigabit
SFI/SFP+ Network Connection (rev 01)
```

##### Steps To Reproduce

```yaml

```

**Comment by vlvkobal** [comment]
`/sys/class/net/<iface>/speed` indicates the interface latest or current speed value, not maximum one. `nic_speed_max` variable is initialized from this value only once together with `net.<interface>` chart. @ktsaou please ask the user to restart netdata and check if the issue is gone after restarting.

**Comment by ktsaou** [comment]
He has restarted it.

This is what he says:

---

The problem is not the old persistent data
I'm sure because completely uninstall netdata and clear install again in /opt.

```
./netdata-uninstaller.sh --force
./netdata-installer.sh --enable-lto --enable-plugin-freeipmi
--dont-wait --install /opt
```

system info:

```sh
# lsb_release -d
Description: Ubuntu 18.04.1 LTS

# cat /sys/class/net/eth2/mtu
9000

# cat /etc/modprobe.d/ixgbe.conf
options ixgbe LRO=1,1,1,1 DCA=2,2,2,2 allow_unsupported_sfp=1,1,1,1

# ethtool -g eth2
Ring parameters for eth2:
Pre-set maximums:
RX: 4096
RX Mini: 0
RX Jumbo: 0
TX: 4096
Current hardware settings:
RX: 4096
RX Mini: 0
RX Jumbo: 0
TX: 4096

# ethtool -i eth2
driver: ixgbe
version: 5.5.1
firmware-version: 0x800000cb
expansion-rom-version:
bus-info: 0000:05:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

0000:05:00.0 = Ethernet controller: Intel Corporation 82599ES
10-Gigabit SFI/SFP+ Network Connection (rev 01)

# cat /proc/net/softnet_stat
26f7e3e5 00000000 0000cf4e 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
252a7fe1 00000000 0000c671 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
27a89dc6 00000000 0000d00f 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
2b01a0c7 00000000 0000d749 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
22ec81e9 00000000 0000bb77 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
1fa25416 00000000 0000ad2a 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
2abaed66 00000000 0000dba2 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
1d60cd00 00000000 00009ff6 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
23c4f753 00000000 0000b5e9 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
1ff362e1 00000000 0000aa1e 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
223f5183 00000000 0000b073 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
28082a9b 00000000 0000c65c 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
266c6d74 00000000 0000bff7 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
22140e9a 00000000 0000b12a 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
28cc99bb 00000000 0000cbc2 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
24fc415d 00000000 0000bb22 00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
```

**Comment by ktsaou** [comment]
Anyway, I think we should refresh this info every few seconds.
Let me do it.

**Comment by ktsaou** [comment]
Found it and fixed it in #4594 
It was a bug.

**Comment by ktsaou** [comment]
merged the fix.
